src/components/donation/Steps/DonateMethods/DonateMethods.tsx

```diff
@@ -12,41 +12,64 @@ type Props = {
   state: FormStep;
 };
 
+const tabIdx = (method?: DonationDetails["method"]) => {
+  switch (method) {
+    case "crypto":
+      return 1;
+    case "paypal":
+      return 4;
+    case "stripe":
+      return 0;
+    default:
+      return 0;
+  }
+};
+
 const tabClasses = (selected: boolean) =>
   `${
     selected
-      ? "border-orange text-orange"
-      : "border-gray-l3 dark:border-bluegray"
-  } uppercase p-2 border-b-4 hover:text-orange-l1 hover:border-orange-l1`;
+      ? "font-semibold bg-blue @md:bg-white text-white @md:text-black"
+      : "border border-prim @md:border-none"
+  } text-sm text-left p-2 @md:p-4 focus:outline-none @md:w-28 rounded @md:rounded-none`;
 
 export default function DonateMethods({ donaterConfig, state }: Props) {
   return (
     <Tab.Group
+      manual
       as="div"
-      className="grid content-start mt-2"
-      defaultIndex={state.details?.method === "crypto" ? 1 : 0}
+      className="grid @md:grid-cols-[auto_1fr]"
+      defaultIndex={tabIdx(state.details?.method)}
```

Comment:
when user goes on with `stripe` and goes back, stripe should still be the selected method